src/components/card/Card.jsx

```diff
@@ -18,22 +31,24 @@ export function Card({ product }) {
         <button
           className="sm-card-fav"
           title={`${
-            productInWishList && user
+            isProductInWishList && user
               ? "Remove From Wishlist"
               : "Add To Wishlist"
           }`}
           onClick={() => {
             if (!user) {
               navigate("/auth");
-            } else if (productInWishList) {
+            } else if (isProductInWishList) {
```

Comment:
This is a great state name with the prefix "is" specifying it as a boolean value. I like this prectice. great work.

Reply:
Thanks 😇